build.gradle

```diff
@@ -13,7 +13,7 @@ import com.synopsys.integration.log.PrintStreamIntLogger
 buildscript {
     ext {
         springBootVersion = '2.2.4.RELEASE'
-        blackDuckCommonVersion = '47.1.0'
+        blackDuckCommonVersion = '47.1.3'
```

Comment:
Is this upgrade required for Cargo support?

Reply:
its required to expose the new forge in integration-bdio